internal/server/config/config_test.go

```diff
@@ -278,7 +278,21 @@ func TestVerifyConfig(t *testing.T) {
 func TestDefaultMaxConditionValuationCost(t *testing.T) {
 	// check to make sure DefaultMaxConditionEvaluationCost never drops below an explicit 100, because
 	// API compatibility can be impacted otherwise
-	require.GreaterOrEqual(t, DefaultMaxConditionEvaluationCost, 100)
+	t.Run("max_condition_evaluation_cost_too_low", func(t *testing.T) {
+		cfg := DefaultConfig()
+		cfg.MaxConditionEvaluationCost = 99
+
+		err := cfg.Verify()
+		require.Error(t, err)
+	})
+	t.Run("max_condition_evaluation_cost_valid", func(t *testing.T) {
+		cfg := DefaultConfig()
+		cfg.MaxConditionEvaluationCost = 120
+
+		err := cfg.Verify()
+		require.NoError(t, err)
+	})
+	require.Equal(t, 120, MaxConditionEvaluationCost()
```

Comment:
```suggestion
	require.Equal(t, 120, MaxConditionEvaluationCost())
```
You're missing a closing `)`. The tests don't build at the moment.

Reply:
Sorry. This is what I get for using the "commit suggestion" button.